opm/parser/eclipse/EclipseState/IOConfig/IOConfig.cpp

```diff
@@ -218,7 +370,7 @@ namespace Opm {
     void IOConfig::initFirstOutput(const Schedule& schedule) {
         m_first_restart_step = -1;
         m_first_rft_step = -1;
-        assertTimeMap( schedule.getTimeMap() );
+        assertTimeMap( this->m_timemap );
```

Comment:
It's really not anymore, but the `DynamicState` will still keep it about. I'll  look into making an extra commit getting rid of it.